src/types/types.tsx

```diff
@@ -1,14 +1,24 @@
-export interface User {
+export interface genericUser {
   id: string;
   access: string;
```

Comment:
I think this field will be obsolete with our new schema as it used to represent the user's type